concent_api/conductor/utils.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict
+from typing import List
+from typing import Union
+
+from conductor.models import BlenderCropScriptParameters
+
+
+def parse_blender_crop_script_parameters_to_dict_from_query(
+        blender_crop_script_parameters: BlenderCropScriptParameters
+) -> Dict[str, Union[int, List[float], bool]]:
+    return dict(
+        resolution=[
+            blender_crop_script_parameters.resolution.x_dimension,
+            blender_crop_script_parameters.resolution.y_dimension,
+        ],
+        borders_x=[
+            float(blender_crop_script_parameters.borders_x.minimum_border_edge),
+            float(blender_crop_script_parameters.borders_x.maximum_border_edge),
```

Comment:
Conversion to `float` is lossy. You're potentially losing precision.

You should either keep Decimals here (if Blender can handle them) or just use `float` in the model (if it can't).

If it's because you can't use Decimals in a RabbitMQ task definition, then just pass them around as strings.

Reply:
Changed conversion to strings